rfcs/20200712-tfrt-kernel-fallback.md

```diff
@@ -0,0 +1,641 @@
+# Title of RFC
+
+| Status        | (Proposed / Accepted / Implemented / Obsolete)          |
+| :------------ | :------------------------------------------------------ |
+| **RFC #**     | [NNN](https://github.com/tensorflow/community/pull/NNN) |
+| **Author(s)** | Anna Revinskaya (annarev@google.com), Jeremy Lau        |
+:               : (lauj@google.com)                                       :
```

Comment:
This syntax does not seem supported by github's markdown renderer

Reply:
Thank you for catching! Fixed. Thats my editor being eager to autowrap and format.